### PR TITLE
Volume snapshots role

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Add rights to manage volume snapshots for admins:
  - Add a new cluster role to grant permissions over volume snapshots and append it to admin adn ns admin roles.
  - This is needed for admins to create and maintain backups for volumes .

**Related issue(s)**
#7862

### CODE REVIEW

File: resources/cluster-users/templates/rbac-roles.yaml [hunk 1]
```diff
@@ -215,6 +215,33 @@ aggregationRule:
...
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-snapshots: "true"
+  annotations:
+    helm.sh/hook-weight: "0"
+rules:
+- apiGroups:
+    - "snapshot.storage.k8s.io"
+  resources:
+    - "volumesnapshots"
+  verbs:
```
Comment (Tomasz-Smelcerz-SAP) [inline resources/cluster-users/templates/rbac-roles.yaml:237]: There is a predefined list of "edit" verbs in values. You can use it here like this:
```
{{ toYaml .Values.clusterRoles.verbs.edit | indent 4 }}
```

Comment (clebs) [reply resources/cluster-users/templates/rbac-roles.yaml:237]: Applied your suggestion, thanks!

**Pull request merged**